src/ubloxGPS.h

```diff
@@ -613,7 +619,7 @@ class ubloxGPS
     // interrupt handler for tx ready signal from gps (if enabled)
     void txReadyHandler();
 
-    void on(void);
+    void on(ubx_dynamic_model_t model = UBX_DEFAULT_MODEL);
```

Comment:
It would be good to start adding some header doc for important interface methods.

Reply:
Added some doxygen comments